lib/ngx/errlog.md

```diff
@@ -210,6 +211,30 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filtering_level
+-----------------------
+**syntax:** *log_level = log_module.get_sys_filtering_level()*
+
+**context:** *any*
+
+Return system default filtering level as an integer. Later the return value
+could be used as Nginx log level constant. For example:
+
+```lua
+init_by_lua_block {
+    local errlog = require "ngx.errlog"
+    local log_level = error_log.get_sys_filtering_level()
+    -- Now the filter level is always one level lower than system default log level
```

Comment:
@agentzh 
Actually I want to say `one level lower` by its integer value...
Obviously my former statement is ambiguous.